app/controllers/admin/link_tracking_controller.rb

```diff
@@ -0,0 +1,34 @@
+# Admin Link Tracking Controller
+class Admin::LinkTrackingController < AdminController
+  before_action :set_dates , only: [:index]
+    
+  # Rendering an index form for link tracking
+  # @author Tracy A. McCormick
+
+  # GET /admin/link_tracking
+  def index
+    stats = Statistics::Base.new({start_date: params[:start_date], end_date: params[:end_date]})
```

Comment:
In this section of code, you are creating a new hash with keyword params and passing in elements from an existing hash.  In actuality, you could just pass `Statistics::Base.new(params)`.

Reply:
updated using params.slice